specification/storagesync/resource-manager/Microsoft.StorageSync/preview/2020-03-01/storagesync.json

```diff
@@ -162,13 +162,39 @@
               "$ref": "#/definitions/StorageSyncService"
             }
           },
+          "202": {
+            "description": "Asynchronous Operation Status Location",
+            "headers": {
```

Comment:
I am not sure whether it is required to list the headers for LRO.
And usually it should still have the schema as final result of LRO.

Reply:
this is consistent with our other definitions

---
In reply to: [404577121](https://github.com/Azure/azure-rest-api-specs/pull/8942#discussion_r404577121) [](ancestors = 404577121)